IPython/lib/display.py

```diff
@@ -89,7 +89,8 @@ class Audio(DisplayObject):
     """
     _read_flags = 'rb'
 
-    def __init__(self, data=None, filename=None, url=None, embed=None, rate=None, autoplay=False, normalize=True):
+    def __init__(self, data=None, filename=None, url=None, embed=None, rate=None, autoplay=False, normalize=True, *,
+                 element_id=None):
```

Comment:
@SylvainCorlay @martinRenou @jasongrout ; is there a conventional name for element_id or is that an acceptable name ? 

I've also made it kwarg only.